docs/instances/instances.md

```diff
@@ -0,0 +1,33 @@
+# Universal Resolver — Instances
+
+Various instances of the Universal Resolver exist that are hosted by DIF as well as the DIF member community.
+
+## DIF instances
+
+### https://dev.uniresolver.io — hosted by DIF
+
+This is the development instance, which runs the latest code-base of the Universal Resolver project.
+
+See [here](/docs/instances/development-instance.md) for more information about this instance.
+
+### https://resolver.identity.foundation — hosted by DIF
+
+This is an alternate name for the same development instance (see above). In the future, this may become a redirect service, or reverse proxy, or load balancer, or similar service that will forward requests to the DIF member instances listed below:
+
+## DIF member instances
```

Comment:
Are these links meant as examples? Since none of them work?

Reply:
Yes, this is still just a proposal with examples.